### DISCUSSION TITLE
osu!lazer crashes on startup on linux

### DISCUSSION THREAD

**Discussion by Aaron-Speedy** [ppy/osu#17849] [Q&A]
### Type

Other

### Bug description

osu!lazer crashes on startup. The window opens but it is black and a few seconds later it crashes into the login screen. I am on Garuda Linux with a Thinkpad x220. 
I have tried with the appimage, the build in the aur, and compiling myself. Every single one of them had the same result. 
I tried reinstalling my operating system(for different purposes, not for osu), and the same thing happened. 

### Version

2022.409.0

### Logs

[database.log](https://github.com/ppy/osu/files/8500929/database.log)
[performance.log](https://github.com/ppy/osu/files/8500930/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8500931/runtime.log)

**Comment by peppy** [comment]
Please consult the manual that came with the operating system. The logs show no issue so it's a native or host level crash. For us to even consider helping, you will need to provide the error log.
